gammapy/spectrum/spectrum_results.py

```diff
@@ -203,6 +203,20 @@ def from_json(data):
 
         return model
 
+    @staticmethod
+    def from_sherpa(covar):
+        """Read sherpa fit output
+
+        Parameters
+        ----------
+        covar : `~sherpa.fit.ErrorEstResults`
+            Fit results as return by :func:`~sherpa.astro.datastack.covar`
+        """
+        model = SpectralModelPowerLaw()
+        model.meta = Bunch(covar.__dict__)
```

Comment:
@joleroi - I don't understand, sorry.

How are you interfacing with Sherpa models at the moment?
Are you sub-classing them to extend them, or do you use factory or converter functions to go back and forth with Gammapy classes?

As I said ... if you have a solution to compute what you need for now, I suggest you run with it, and then I'll read over and meditate about it in the coming days ...